src/Dynamo.All.sln

```diff
@@ -223,296 +223,594 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoPackagesWPF", "Dynamo
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoWPFCLI", "DynamoWPFCLI\DynamoWPFCLI.csproj", "{F7FD9395-35D5-474D-8AD1-9904817E70D3}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Extensions", "Extensions", "{88D45B00-E564-41DB-B57C-9509646CAA49}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PackageDependencyViewExtension", "PackageDependencyViewExtension\PackageDependencyViewExtension.csproj", "{5E76AAB3-6302-473E-9655-081B53FB1419}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
 		Release|Any CPU = Release|Any CPU
+		Release|x64 = Release|x64
```

Comment:
why do we need these x64 configs?

Reply:
I had the same question but tested these are auto generated once I move all the extension projects to extension folder without even touching and project properties, visual studio magic

Reply:
I am not convinced that is a good thing.

Reply:
I dont think this is a bad thing either, for each individual project, the configuration and platform is stored in csproj, so there is no change there and those settings still dominate. The particular `GlobalSection` of `preSolution` defines the loading order, this is also auotgenerated from VS2017 not VS2019.